inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -0,0 +1,131 @@
+"""Coastal area scenario."""
+
+from functools import singledispatchmethod
+import math
+import os
+from typing import Optional
+import shutil
+from uuid import UUID
+
+from inductiva.types import Path
```

Comment:
As you know, I am not a big fun of importing classes, since it reduces readability down in the code.
wdyt @hpenedones 

I would prefer a simple:

```
import inductiva
```

or maybe have one special type of alias that we use consistently as:
```
import from inductiva.utils.templates as T
```
this is similar to what is done in Pytorch and TF.

Reply:
This is done across the client package and demo/example scripts, so I think this deserves an housekeeping issue to be tackled throughout.